android/src/main/java/com/dylanvann/fastimage/FastImageViewConverter.java

```diff
@@ -104,6 +105,7 @@ static RequestOptions getOptions(ReadableMap source) {
         }
         return new RequestOptions()
                 .diskCacheStrategy(diskCacheStrategy)
+                .override(Target.SIZE_ORIGINAL)
```

Comment:
Beware that this line makes android crash when huge images are being loaded as no compression will take place
```
.override(Target.SIZE_ORIGINAL)
```